build.gradle.kts

```diff
@@ -15,12 +16,19 @@ dependencies {
     implementation(kotlin("stdlib"))
     implementation("io.ktor:ktor-client-core:${Versions.KTOR}")
     implementation("io.ktor:ktor-client-cio:${Versions.KTOR}")
-    implementation("io.ktor:ktor-client-serialization:${Versions.KTOR}")
+    implementation("com.squareup.moshi:moshi:${Versions.MOSHI}")
```

Comment:
why not kotlinx serialization?
Android implementation: https://github.com/getkevin/kevin-android/blob/master/core/src/main/java/eu/kevin/core/networking/BaseApiFactory.kt

Reply:
Kotlinx.serialization is still unstable (https://kotlin.github.io/kotlinx.serialization/kotlinx-serialization-core/kotlinx-serialization-core/kotlinx.serialization/-serializer/index.html). Replacing the serializer should be easy though

Reply:
But it is stable already since October last year.
https://github.com/Kotlin/kotlinx.serialization/releases